src/Raven.Server/ServerWide/Commands/ETL/UpdateEtlCommand.cs

```diff
@@ -156,4 +160,45 @@ public override void UpdateDatabaseRecord(DatabaseRecord record, long etag)
             new AddSnowflakeEtlCommand(Configuration, DatabaseName, null).UpdateDatabaseRecord(record, etag);
         }
     }
+
+    public sealed class UpdateAiEtlCommand : UpdateEtlCommand<AiEtlConfiguration, AiConnectionString>
+    {
+        public UpdateAiEtlCommand()
+        {
+            // for deserialization
+        }
+
+        public override void UpdateDatabaseRecord(DatabaseRecord record, long etag)
+        {
+            var oldConfig = record.AiEtls.FirstOrDefault(x => x.Name == Configuration.Name);
```

Comment:
shouldn't all of the validation be in the endpoint? @ArieSLV check with Karmel

Reply:
Done.